applications/crossbar/priv/couchdb/schemas/cdrs.json

```diff
@@ -136,6 +168,14 @@
             "description": "correlating ID among related call legs",
             "type": "string"
         },
+        "interaction_key": {
+            "description": "",
```

Comment:
Need descriptions

Reply:
Yeah, I don't know what they mean, so I don't know how to describe them ;)